content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -169,130 +169,159 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
-  -NetworkMode overlay `
-  -ClusterCIDR <Cluster CIDR> `
-  -ServiceCIDR <Service CIDR> `
-  -KubeDnsServiceIP <Kube-dns Service IP> `
-  -LogDir <Log directory>
-```
-
-| Parameter | Default Value | Notes |
-| --- | --- | --- |
-| -ManagementIP | N/A (required) | The IP address assigned to the Windows node. You can use `ipconfig` to find this. |
-| -NetworkMode | l2bridge | We're using `overlay` here |
-| -ClusterCIDR | 10.244.0.0/16 | Refer to your cluster IP plan |
-| -ServiceCIDR | 10.96.0.0/12 | Refer to your cluster IP plan |
-| -KubeDnsServiceIP | 10.96.0.10 | |
-| -InterfaceName | Ethernet | The name of the network interface of the Windows host. You can use <code>ipconfig</code> to find this. |
-| -LogDir | C:\k | The directory where kubelet and kube-proxy logs are redirected into their respective output files. |
+This step will perform the following actions:
+  1. Connect to the control-plane ("Master") node via SSH, to retrieve the [Kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file.
+  1. Register kubelet as a Windows service
+  1. Configure CNI network plugins
+  1. Create a HNS network on top of the chosen network interface
+      {{< note >}}
+      This may cause a network blip for a few seconds while the vSwitch is being created.
+      {{< /note >}}
+  1. (If vxlan plugin is selected) Open up inbound firewall UDP port 4789 for overlay traffic
+  1. Register flanneld as a Windows service
+  1. Register kube-proxy as a Windows service
 
 Now you can view the Windows nodes in your cluster by running the following:
 
 ```bash
 kubectl get nodes
 ```
 
-{{< note >}}
-You may want to configure your Windows node components like kubelet and kube-proxy to run as services. View the services and background processes section under [troubleshooting](#troubleshooting) for additional instructions. Once you are running the node components as services, collecting logs becomes an important part of troubleshooting. View the [gathering logs](https://github.com/kubernetes/community/blob/master/sig-windows/CONTRIBUTING.md#gathering-logs) section of the contributing guide for further instructions.
-{{< /note >}}
+#### Remove the Windows Node from the Kubernetes cluster
+In this section we'll cover how to remove a Windows node from a Kubernetes cluster. 
+
+Use the previously downloaded [KubeCluster.ps1](https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/kubeadm/v1.15.0/KubeCluster.ps1) script to remove the Windows node from the cluster:
+
+  ```PowerShell
+    .\KubeCluster.ps1 -ConfigFile .\Kubeclustervxlan.json -reset 
+  ```
+  where `-ConfigFile` points to the path of the Kubernetes configuration file.
+
+![alt_text](../kubecluster.ps1-reset.gif "KubeCluster.ps1 reset output")
+
+This step will perform the following actions on the targeted Windows node:
+  1. Delete the Windows node from the Kubernetes cluster
+  1. Stop all running containers and remove all container images
```

Comment:
I don't believe we remove all container images

Reply:
Hm not sure why I was seeing that but I updated this to just say "Stop all running containers" now (I verified in the script that we still do that by running `docker rm` on all the containers). Thanks!